docs/Tutorials/Visualization.md

```diff
@@ -40,18 +40,20 @@ volume data will be described in the next section.
 A SpECTRE executable with observers produces volume and/or reduced data h5
 files. An XDMF file must be created from the volume data in order to do
 visualization using ParaView. To this end we provide the python executable
-`GenerateXdmf.py` in the `tools` directory. The `--file-prefix` and `--output`
-arguments must be specified when the command is run; they correspond to the
-names of the H5 volume data file and the desired .xmf file name, respectively,
-and without filename extensions. Leave out the node number when providing the
-name of the volume data file, these will be appended by `GenerateXDMF.py`.
-Use `--help` to see a further description of possible arguments. Open the .xmf
-file in ParaView and select the `Xdmf Reader`, *not* the version 3 readers. On
-the left hand side of the main ParaView window is a section named `Properties`,
-here you must click the highlighted `Apply` button. ParaView will now render
-your volume data. If you only wish to visualize a few datasets out of a large
-set, we recommended unchecking the boxes for the datasets you wish to ignore
-under `Point Arrays` before clicking `Apply`.
+`GenerateXdmf.py` in the `tools` directory. `GenerateXdmf.py` takes two
+arguments which are passed to --file-prefix` and `--output`. The argument
+passed to `--file-prefix` is the name of the H5 volume data file, without
+the filename extension. Leave out the node number when providing the name
+of the volume data file, these will be appended by `GenerateXDMF.py`. The
+argument passed to `--output` is the desired .xmf file name, also without
```

Comment:
I would say "... name of the H5 volume data file, leaving out the node number and extension. The argument passed to --output..."

Reply:
@wthrowe @nilsdeppe please look at my suggested wording

Reply:
:+1: sounds good to me

Reply:
I've made this change :+1: